bigquery/README.md

```diff
@@ -2,32 +2,35 @@
 
 ### Quickstart
 
-Index a BigQuery table into an Elasticsearch container on your local machine.
+* If `~/.config/gcloud/application_default_credentials.json` doesn't exist,
+create it by running `gcloud auth application-default login`.
+* Providing a GCP Project ID that you can use as the billing project, run the
```

Comment:
* Determine the project that will be billed for querying the BigQuery tables. You must have `bigquery.jobs.create` on this project; for example, any project where you are Viewer/Editor/Owner.
* Run indexer: `BILLING_PROJECT_ID=<billing_project_id> docker-compose up --build`

Same below.

Reply:
Done